Author: Niko Tyni <ntyni@debian.org>
Subject: APACHE_TEST_EXTRA
 Honour $(APACHE_TEST_EXTRA_ARGS); Makefile.PL has apparently not been updated
 since this was added to Apache::Test.
Forwarded: not-needed
Last-Update: 2011-03-22
Reviewed-by: Nicholas Bamber <nicholas@periapt.co.uk>
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -783,12 +783,12 @@
 
 test_clean :
 	$(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) \
-	t/TEST -clean
+	t/TEST $(APACHE_TEST_EXTRA_ARGS) -clean
 
 run_tests : test_clean
 	$(PASSENV) \
 	$(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) \
-	t/TEST -bugreport -verbose=$(TEST_VERBOSE) $(TEST_FILES)
+	t/TEST $(APACHE_TEST_EXTRA_ARGS) -bugreport -verbose=$(TEST_VERBOSE) $(TEST_FILES)
 	$(MAKE) -k run_subtests
 
 run_subtests ::
